### PR TITLE
yarrn mappings

### PR DESCRIPTION
**Pull request by sylv256**
arr

### CODE REVIEW

File: mappings/net/minecraft/block/Block.mapping [hunk 1]
```diff
@@ -52,6 +52,9 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 	METHOD method_9503 getBlockFromItem (Lnet/minecraft/class_1792;)Lnet/minecraft/class_2248;
 		ARG 0 item
 	METHOD method_9504 rainTick (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)V
+		ARG 1 state
```
Comment (i509VCB) [inline mappings/net/minecraft/block/Block.mapping:55]: This has been covered by atleast 3 other PRs. I would probably leave these params.

File: mappings/net/minecraft/class_5493.mapping [hunk 2]
```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_5493
```
Comment (i509VCB) [inline mappings/net/minecraft/class_5493.mapping]: A name would be nice for this.

File: mappings/net/minecraft/class_5562.mapping [hunk 3]
```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_5562
```
Comment (i509VCB) [inline mappings/net/minecraft/class_5562.mapping:1]: A name would be nice for this.

File: mappings/net/minecraft/class_5584.mapping [hunk 4]
```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_5584
```
Comment (i509VCB) [inline mappings/net/minecraft/class_5584.mapping]: A name would be nice for this.

File: mappings/net/minecraft/class_5624.mapping [hunk 5]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_5624
```
Comment (i509VCB) [inline mappings/net/minecraft/class_5624.mapping:1]: A name would be nice for this.

File: mappings/net/minecraft/client/render/block/entity/BlockEntityRendererFactories.mapping [hunk 6]
```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_5616 net/minecraft/client/render/block/entity/BlockEntityRendererFactories
```
Comment (i509VCB) [inline mappings/net/minecraft/client/render/block/entity/BlockEntityRendererFactories.mapping]: Already covered in another PR?

File: mappings/net/minecraft/entity/Entity.mapping [hunk 7]
```diff
@@ -684,3 +704,25 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 			ARG 2 x
 			ARG 4 y
 			ARG 6 z
+	CLASS class_5529 RemovalReason
```
Comment (i509VCB) [inline mappings/net/minecraft/entity/Entity.mapping]: Another thing that another pr has covered.

**Review by i509VCB — CHANGES_REQUESTED** [review]
Tehc you did put in effort but I believe that in future PRs you need to ask what is being done and pick something specific you can cover without stepping over others

**Comment by sylv256** [comment]
lmao i deleted all of the stuff

**Comment by YanisBft** [comment]
Your PR doesn't make any sense

**Pull request closed without merging**